.github/actions/setup-flutter/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Setup 
+description: Setup runner to be able to build an app with Flutter. Not specific for Android or iOS.
+
+runs:
+  using: "composite"
+  steps:
+  - uses: subosito/flutter-action@v2
+    with:
+      flutter-version: '3.0.5'
+      channel: 'stable'
+      cache: true
+
+  - name: Install Flutter dependencies
+    run: flutter pub get
+    shell: bash 
+
+  # flutter analyze throws error with this file missing. 
+  # In future CI work for compiling sample apps, we can move this logic into another github workflow file with populating real credentials.
+  - name: Create credentials.env file for sample app 
+    run: cp example/credentials.env.example example/credentials.env
```

Comment:
We don't use this file anymore in `example`, the fact `credentials.env` still lives in repo is probably why its causing this issue.